.github/actions/publish-npm/action.yml

```diff
@@ -17,4 +17,4 @@ runs:
         ./scripts/publish-npm.sh
       env:
         LD_RELEASE_IS_PRERELEASE: ${{ inputs.prerelease }}
-        LD_RELEASE_IS_DRYRUN: ${{ inputs.dry_run }}
+        LD_RELEASE_IS_DRYRUN: ${{ inputs.dry-run }}
```

Comment:
It seems like without specifying it would default to true and was skipping the actual release.